docker doesn't start: `Authentication Incorrect for reason: basic_auth(admin)`

**Describe the bug**
I can't change defalut `TERMINUSDB_ADMIN_PASS=root` on setup with Docker.
```sh
docker compose up
```

Output:
```sh
terminusdb-server                | [ERROR] 2023-12-01T22:21:26.896009+00:00 Authentication Incorrect for reason: basic_auth(admin)
terminusdb-server                | 
terminusdb-server                | [INFO] 2023-12-01T22:21:26.900388+00:00 POST /api/organizations/terminusCR (401)
terminusdb-change-request-api-1  | An unexpected error occured during startup
terminusdb-change-request-api-1  | Error: API Error @type api:ErrorResponse api:message Incorrect authentication information api:status api:failure Code: 401 url: http://terminusdb-server:6363/api/organizations/terminusCR
terminusdb-change-request-api-1  |     at Object.apiErrorFormatted (/app/node_modules/@terminusdb/terminusdb-client/lib/errorMessage.js:95:13)
terminusdb-change-request-api-1  |     at /app/node_modules/@terminusdb/terminusdb-client/lib/dispatchRequest.js:227:30
terminusdb-change-request-api-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
terminusdb-change-request-api-1  |   data: {
terminusdb-change-request-api-1  |     '@type': 'api:ErrorResponse',
terminusdb-change-request-api-1  |     'api:error': { '@type': 'api:IncorrectAuthenticationError' },
terminusdb-change-request-api-1  |     'api:message': 'Incorrect authentication information',
terminusdb-change-request-api-1  |     'api:status': 'api:failure'
terminusdb-change-request-api-1  |   },
terminusdb-change-request-api-1  |   status: 401
terminusdb-change-request-api-1  | }
terminusdb-change-request-api-1 exited with code 1

```

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone https://github.com/terminusdb/terminusdb`
2. `cd terminusdb`
3. `touch .env`
4. edit `docker-compose.yml` and set `services` → `change-request-api` → `environment` → `TERMINUSDB_ADMIN_PASS` and `services` → `terminusdb-server` → `environment` → `TERMINUSDB_ADMIN_PASS` to `my-strong-password-123`
5. `docker compose up`


**Expected behavior**
terminusdb docker containers start without errors

**Screenshots**
—

**Info (please complete the following information):**
terminusdb version: [`v11.1.10`](https://github.com/terminusdb/terminusdb/releases/tag/v11.1.10)

**Additional context**
If the `docker-compose.yml` file is not edited (with `TERMINUSDB_ADMIN_PASS` disabled), Docker containers start correctly (but with unsecure `root` password!)

--- a/.env-template
+++ b/.env-template
@@ -1,0 +1,2 @@
+# Add a password after the =
+TERMINUSDB_ADMIN_PASS=

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,5 +1,12 @@
-version: "3"
-
+# Instructions:
+#
+# 1. Copy .env-template to .env
+# 2. Add your admin password to .env
+# 3. docker compose up -d
+# 4. docker compose logs (to verify all started)
+# 5. connect to localhost:6363, login with admin and the .env password
+#
+ 
 services:
   terminusdb-server:
     image: terminusdb/terminusdb-server:latest
@@ -18,8 +25,8 @@ services:
       - TERMINUSDB_INSECURE_USER_HEADER_ENABLED=true
 
       # Admin-Password should be changed before exposing to the internet:
-      # (it's recommended to use the .env)
-      # -TERMINUSDB_ADMIN_PASS=root
+      # (Create a .env file from .env-template and add a password)
+      - "TERMINUSDB_ADMIN_PASS=${TERMINUSDB_ADMIN_PASS:?error}"
     volumes:
       # For the use of a local dashboard
       #      - ./dashboard:/app/terminusdb/dashboard
@@ -44,15 +51,18 @@ services:
       - USE_CHANGE_REQUEST=${USE_CHANGE_REQUEST:-true}
       # Add your OpenAI key in a .env file
       - OPENAI_SERVER_URL=http://vectorlink:8080
+      - "USER_KEY=${TERMINUSDB_ADMIN_PASS:?error}"
 
       # There are multiple ways to configure TerminusDB security through
       # environment variables. Several reasonable options are included below.
       # Uncomment the option you decide on and comment out others.
-      # Don't forget to change the default password!
+      # Don't forget to change the default password in the .env file
 
       # TerminusDB should be set up behind a TLS-terminating reverse
       # proxy with admin authentication provided by password.
-      # - USER_KEY=root  #  Change before exposing to the internet.
 
       # The storage path of terminusdb databases is /app/terminusdb/storage in case
       # you want to persist storage somewhere else.
+
+      # To change the terminusdb password if already set, to match the .env:
+      # docker compose exec -ti terminusdb-server ./terminusdb user password admin -p [FancyPassword]

